client/src/components/NavDrawerCloseContext.tsx

```diff
@@ -0,0 +1,7 @@
+import { createContext } from 'react';
+
+export const closeContext = createContext({
```

Comment:
Although it would seem that `closeContext` should be named like a variable, React Context is a special case since you render it directly in JSX. Context variable names should start with uppercase.